components/VFBMain.js

```diff
@@ -593,10 +585,7 @@ export default class VFBMain extends React.Component {
 
   handleClickOutside () {
     if (this.toolBarRef && !this.toolBarRef.contains(event.target)) {
-      this.setState({
-        UIUpdated: true,
```

Comment:
this should make it back since it's used for the ShouldComponentUpdate to minimise the not needed re-render